ansible/roles/kind/kind_cluster_create/meta/main.yml

```diff
@@ -0,0 +1,53 @@
+galaxy_info:
```

Comment:
Do you plan to fill galaxy_info in metafiles for all roles? I think these files are not required when roles used locally and can be safely dropped if we don't plan to upload roles to the ansible galaxy.

Reply:
no, I removed it